src/nifgen/metadata/functions.py

```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-# This file is generated from NI-FGEN API metadata version 21.5.0d106
+# This file is generated from NI-FGEN API metadata version 22.0.0d9999
```

Comment:
Let's try to grab this from a real internal NI build artifact.
If you cannot, then let's temporarily override the name through the add-on.

Reply:
Updated